### ISSUE TITLE
parse error at line 4, column 20, token: Int 2

### ISSUE THREAD

**Issue by LOuroboros** [mtolly/pokemid#19]
I'm not sure if I made a mistake, but I tried to convert the music of Pallet Town directly obtained from Pokered using this tool, and it's not working.

I copied the contents of [pallettown.asm](https://raw.githubusercontent.com/pret/pokered/master/audio/music/pallettown.asm) into an individual .asm file _(because I'm too lazy to clone the entire repository to get a single track)_ and pasted it in a "pallet.asm" file on my desktop.
Then I downloaded the latest win32 compiled build from the Releases page, used the tool as the Readme instructs to, and this is what I got:
![Untitled898](https://user-images.githubusercontent.com/4485172/98446619-66e48880-20fd-11eb-9986-d76dea3b2fb8.png)

**Comment by ShadyGuyJose** [comment]
I had the same issue today. This hasn't been updated in years, and I noticed that the readme mentioned a different syntax to what I saw in the code, so I had the idea of looking for an older fork of pokered. I tried with the "rgbgfx" branch version of an audio file and it worked. This tool has to be updated for whatever new syntax/language is being used.

**Comment by mtolly** [comment]
Yeah, what @ShadyGuyJose said, the disassembly project has changed the macros used for the music. I'll see if it is an easy fix to update it to the current version.

**Comment by andrewmcwatters** [comment]
Hey @mtolly, thanks for keeping this around on GitHub. At [Planimeter](https://www.planimeter.org), we've been using pokemid for testing the [Grid Engine](https://www.planimeter.org/grid-sdk/) for years. It's helped us as a part of establishing a baseline development experience when playing back Pokémon Red levels converted to a modern 2D game engine. We use extracted MIDI files produced from your work to simulate soundscapes in internal builds.

![](https://i.imgur.com/UtzSbjy.png)

**Comment by mtolly** [comment]
@andrewmcwatters That's awesome! Always love hearing about cool stuff people make with things I built :grinning: 

@LOuroboros @ShadyGuyJose The version `0.9` I just put up should work with the latest `pokered` and `pokeyellow` repositories; I tested a round-trip conversion of all the music files (minus two with special labels referred to elsewhere) and the games recompiled without error, and the title screen sounded fine. Let me know if there's any subtler problems I missed!
